agent_build_refactored/tools/runner.py

```diff
@@ -831,7 +830,7 @@ def run_in_docker(
 
             mount_args.extend([
                 "-v",
-                f"{arg.path}:{in_docker_path}:z"
```

Comment:
Why is the :z no longer necessary?  It's no longer shared?

Reply:
I, actually, haven't got any difference with or without it. I just added it in hope that it will solve user permission differences of host and guest OS's, but it didn't.